### DISCUSSION TITLE
Use vuetify-dialog for simple dialogs

### DISCUSSION THREAD

**Discussion by ferferga** [jellyfin/jellyfin-vue#718] [Questions] [answered]
What about using a library that provides simple dialogs for actions like OK, delete and such? I propose vuetify-dialog: https://github.com/yariksav/vuetify-dialog

**Comment by ThibaultNocchi** [comment]
Yeah I could see that, does it use vuetify modals?

**Reply by ferferga** [reply]
Yes, it's just a wrapper that use preconfigured Vuetify components programatically

**Reply by ThibaultNocchi** [reply]
Yeah then this could be helpful!

**Comment by ferferga** [comment, marked as answer]
As of now, we have ``ConfirmDialog``, invokable through the ``useConfirmDialog`` composable
